test/verify/netlib.py

```diff
@@ -55,6 +55,13 @@ def nm_activate_eth(self, iface):
         m.execute("nmcli con up %s ifname %s" % (iface, iface))
         self.addCleanup(m.execute, "nmcli con delete %s" % iface)
 
+    def nm_login(self, disable_checkpoints=True, checkpoint_settle_time=None):
+        self.login_and_go("/network")
```

Comment:
I'd rather this class would just override `login_and_go`. This avoids having to touch all tests, and more importantly, we won't forget in the future that there is a special method.

There could just be a method `self.enable_settle_test()` or so which does the variable poking, or even a decorator.

Reply:
> I'd rather this class would just override login_and_go

Good idea, I didn't even think that far...

Reply:
Done.